examples/azureml/Dockerfile.dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM mcr.microsoft.com/azureml/aifx/stable-ubuntu2004-cu115-py38-torch1110
+
+USER root:root
+
+RUN pip install pybind11
+
+RUN pip install git+https://github.com/microsoft/DeepSpeed.git@azure
```

Comment:
@msp8955 -- you can use the public DeepSpeed master branch and no longer need the azure branch.